frontend/scripts/build

```diff
@@ -20,6 +20,7 @@ yarn install || exit 1;
 rm -rf resources/public;
 rm -rf target/dist;
 
+yarn run build:renderer
```

Comment:
This is a experiment code that is not going to go to be deployed right now until we have something usable. 

We shouldn't penalize every frontend build with a module that is only going to be used locally for the moment.